interfaces/apparmor/spec.go

```diff
@@ -88,6 +90,65 @@ func RegisteredSnippetKeys() []string {
 	return keylist
 }
 
+// MetadataTag is an opaque string used to tag a snippet.
+//
+// Metadata tags can be added to AppArmor rules or snippets to encode additional
+// information which the kernel can then send back to snapd when it sends a
```

Comment:
I hope I understood the intent correctly but even if not please clarify the `it` I've replaced.

```suggestion
// information which the kernel can then send back to snapd when the kernel sends a
```